gef.py

```diff
@@ -7277,15 +7286,16 @@ def __init__(self) -> None:
         super().__init__(complete=gdb.COMPLETE_LOCATION)
         return
 
+    @parse_arguments({"arena_address": ""}, {})
     @only_if_gdb_running
-    def do_invoke(self, argv: List[str]) -> None:
+    def do_invoke(self, *_: Any, **kwargs: Any) -> None:
+        args = kwargs["arguments"]
         if gef.heap.main_arena is None:
-            err("Invalid Glibc arena")
+            err("Heap not initialized")
             return
-
-        arena_addr = f"*{argv[0]}" if len(argv) == 1 else gef.heap.selected_arena
-        gef_print(titlify(f"Unsorted Bin for arena '{arena_addr!s}'"))
-        nb_chunk = GlibcHeapBinsCommand.pprint_bin(arena_addr, 0, "unsorted_")
+        arena_addr = args.arena_address if args.arena_address else gef.heap.selected_arena.addr
```

Comment:
Same, better keep `arena_addr` as an int by `parse_address` first `args.arena_address`

```suggestion
        arena_addr = parse_address(args.arena_address) if args.arena_address else gef.heap.selected_arena.addr
```